entries/entry04.md

```diff
@@ -1,7 +1,264 @@
-# Entry 4
-##### X/X/XX
+# Entry 04
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 1

Great job! I love the video.